### PR TITLE
Merging "final" branch which contains refactored code

### PR DESCRIPTION
**Pull request by orchidenver**
1. Accidentely pushed to git .env file is no longer tracked (added to gitignore);
2. Allocated theme into separate 'theme' folder. 
3. Screen view isolated into separate page ScreenView.tsx
4. Name of the folder 'interfaces' changed to 'types'. Types had been reorganized.
5. Code was slightly refactored.

### CODE REVIEW

File: src/helpers/index.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  forecastData: CurrentWeatherType[],
+  days: number = 0
+) {
+  const day: Date = new Date();
+  const endOfADay: number =
+    day.setUTCHours(23, 59, 59, 999) + days * 1000 * 60 * 60 * 24;
+
+  const data = forecastData
+    ?.filter((today: CurrentWeatherType, i: number) => {
+      return +today.date * 1000 < endOfADay;
```
Comment (ivan-ivan) [inline src/helpers/index.tsx]: Usually more explicit ways of converting to number type preferred such as `Number` or `parseInt`

File: src/helpers/index.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    })
+    .map((el: CurrentWeatherType) => {
+      return {
+        ...el,
+        date: +el.date * 1000,
```
Comment (ivan-ivan) [inline src/helpers/index.tsx]: Usually more explicit ways of converting to number type preferred such as `Number` or `parseInt`

**Review by ivan-ivan — APPROVED** [review]
Looks good! Just left a minor comment

File: src/helpers/index.tsx (hunk 1 above)
Comment (orchidenver) [reply src/helpers/index.tsx]: Got it. I replaced unari plus with parseInt.

File: src/helpers/index.tsx (hunk 2 above)
Comment (orchidenver) [reply src/helpers/index.tsx]: > Usually more explicit ways of converting to number type preferred such as `Number` or `parseInt`

Got it. I replaced unari plus with parseInt.

File: src/helpers/index.tsx (hunk 1 above)
Comment (orchidenver) [reply src/helpers/index.tsx]: > Usually more explicit ways of converting to number type preferred such as `Number` or `parseInt`

Got it. I replaced unari plus with parseInt.

**Pull request merged**